cmd/etcd-launcher/main.go

```diff
@@ -158,6 +146,23 @@ func main() {
 		log.Panicw("join cluster as fresh member", zap.Error(err))
 	}
 
+	// setup and start etcd command
+	etcdCmd, err := startEtcdCmd(e, log)
+	if err != nil {
+		log.Panicw("start etcd cmd", zap.Error(err))
+	}
+
+	if err = wait.Poll(1*time.Second, 60*time.Second, func() (bool, error) {
+		return e.isClusterHealthy(log)
+	}); err != nil {
+		log.Panicw("manager thread failed to connect to cluster", zap.Error(err))
+	}
+
+	// handle changes to peerURLs (https -> http)
+	if err := e.updatePeerURLs(log); err != nil {
+		log.Panicw("failed to update peerURL", zap.Error(err))
+	}
```

Comment:
Why is this being called here? The `https -> http` change is no longer true since `etcd-launcher` now supports peer TLS connections. `updatePeerURLs()` is already being called earlier in the code.